### ISSUE TITLE
neat/grid/_media.scss makes gulp-sass crash

### ISSUE THREAD

**Issue by suitcasemedia** [thoughtbot/neat#396]
We've been working with Neat framework for 6 months and love it.
However suddenly we have a critical problem with one of your excellent mixins which we are entirely dependant on for media queries. 
Here's a link to  the file.

https://github.com/thoughtbot/neat/blob/master/app/assets/stylesheets/grid/_media.scss

Heres the docs for the mixin.

http://thoughtbot.github.io/neat-docs/latest/#media

We've called it in our sass files almost 300 times so its pretty essential, although we mostly only use the first parameters e.g.  media( max-width $small-screen)
We also used it in a few of your navigation modules from refills.

Don't know what the error is because gulp-sass and browser-sync crash without sending error messages to the console like it normally does with the way we have it set up.They only crash when  this mixin is loaded. Don't know what causes the error

If I start removing  the mixin we get error reporting back and gulp-sass doesn't crash any more. ALthough we get errors from all the files that call the mixin.

Does anyone have any suggestions/ ideas / solutions ?

**Comment by suitcasemedia** [comment]
I tried running the code through the ruby sass incase the problem is with gulp.

It worked better and gave different errors…

Is the problem with gulp ?

**Comment by suitcasemedia** [comment]
Ok now I have an error message hooraye! error neat/grid/_media.scss (Line 87: Illegal nesting: Only properties maybe nested beneath properties)

Does anyone have any idea what this means?

**Comment by suitcasemedia** [comment]
Ok so there are no syntax errors from _media.scss as I can see here
http://sassmeister.com/gist/52cd587b351f13c35e81

Where on earth could this error be coming from?

**Comment by suitcasemedia** [comment]
Why would the compiler point to this file?

**Comment by suitcasemedia** [comment]
OK this was my fault - not the mixins. Its because of the way I was calling it.
 I noticed it worked when I commented out lines which have the '@content ' directives:

@content

These directives are basically just place holders for me(the end user) to put my sass/css that I want in the mixin when I am calling it from another sass file. This is the general css rules that I want applied inside a certain width. It was my own files else where that were breaking the mixin and making gulp crash.

I commented out all 60 of the sass files that were calling the mixin one by one until the pipe from Sass to Css didn’t break any more.

The file that caused the problem happens to be called _search-drawers.sass

If you looked in the .find-drawer class you would have  seen that all the media queries are nested inside the margin-bottom: -33px rule:

.find-drawer
  margin-bottom: -33px
    @include media(max-width $tablet-size)
      margin-bottom: 0
    @include media(max-width $small-screen)
      margin-bottom: 0
    @include media(max-width $mobile-screen)
      margin-bottom: 0
This must have generated a really crazy media queries I would love to see what it actually was generating before it crashed

I just shifted the media queries a couple of spaces to the left so that they are now nested inside .find-drawer class

.find-drawer
  margin-bottom: -33px
  @include media(max-width $tablet-size)
    margin-bottom: 0
  @include media(max-width $small-screen)
    margin-bottom: 0
  @include media(max-width $mobile-screen)
    margin-bottom: 0
Phew! What a relief !

Thanks to the people who made this lovely framework...
